app/helpers/application_helper.rb

```diff
@@ -1,3 +1,16 @@
 module ApplicationHelper
-
+  # Helper to yield common CSS classes for lists
+  # Iterates through the given collection and yields three arguments for each member:
+  #  * The member
+  #  * An array of classes including either "odd" or "even", and "first"/"last" as relevant
+  #  * The index of this member
+  def each_with_list_classes(collection, &block)
```

Comment:
Unused method argument - `block`. If it's necessary, use `_` or `_block` as an argument name to indicate that it won't be used.